Sephora-M/assignment4/MapTest.java

```diff
@@ -0,0 +1,69 @@
+package assignment4;
+
+import static org.junit.Assert.*;
+
+import org.junit.Test;
+
+public class MapTest {
```

Comment:
Very nice test cases! The only thing I could add is that you might want to also unit test some of the helper methods, like getNeighbors for example.

Reply:
Thanks! Will do 👍

Reply:
Actually I have a question; is it common to test private methods? If so, is there a way around having to make them public?

Reply:
Usually you make them package-protected (package only visible), and possibly mark as `@VisibleForTesting` (Guava supplies this annotation).